CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unreleased
+
+* Allow summary list custom text ([PR #1483](https://github.com/alphagov/govuk_publishing_components/pull/1483))
```

Comment:
Andy would've said: shall we align this with the PR title?

Reply:
He would, too. Darn him. Okay, I'll fix it...